docs/cloudlinux-os-plus/README.md

```diff
@@ -1556,16 +1573,78 @@ To disable both suites:
 cloudlinux-awp-admin set-suite --suite=accelerate_wp,accelerate_wp_premium --disallowed --users=<username>
 ```
 
+### WHMCS billing
+
+CloudLinux developed it's own WHMCS plugin 
+which provides you AccelerateWP billing integration out of the box. 
+Check out [the documentation](WHMCS_DOCS_URL_HERE) to find how to install and use the plugin.
+
+### Other billing integration
+
+As AccelerateWP Premium is the feature that works on subscription basis, 
+we made it possible for hosters to integrate their existing billing systems with our plugin and sell the 
+feature for their users.
+
+When AccelerateWP Premium is enabled in admin interface, users get proposal to upgrade their subscription.
+
+![](./images/awp/AWPUpgradeNoLink.png)
+
+When user upgrades the subscription to the plan with AccelerateWP support, 
+billing must execute the following command on the server:
+```
+cloudlinux-awp-admin set-suite --suite=accelerate_wp_premium --allowed --users=<username>
+```
+
+This command makes two things:
+- allow user to install plugin, bypassing the upgrade window;
+- reports premium as being used to CLN, starting the billing cycle for hoster.
+
+
+When user terminates or downgrades plan, the following command must be executed by the billing system:
+```
+cloudlinux-awp-admin set-suite --suite=accelerate_wp_premium --default --users=<username>
+```
+
+### Setup upgrade URL for AccelerateWP Premium
+
+The upgrade window can be customized with link to the plan upgrade page, which can be set using CLI command:
+
+```
+cloudlinux-awp-admin set-options --upgrade-url https://plan.upgrade/splash
+```
+
+![](./images/awp/AWPUpgradeLink.png)
+
+
+In order to remove the link, just set upgrade url to the empty one.
+
+```
+cloudlinux-awp-admin set-options --upgrade-url ''
+```
+
+### Disable AccelerateWP Premium
+
+If you would like to stop using AccelerateWP Premium, 
+click on the `manage` link and remove `Premium Features` checkbox.
+AccelerateWP will be still available for your users.
+
+![](./images/awp/AWPDisablePremiumOnly.png)
+
 ### Disable AccelerateWP
-If you would like to stop using AccelerateWP, toggle the `Enable AccelerateWP for all users` back.
 
-![](/images/AWPDisable.png)
+If you would like to stop using AccelerateWP completely, toggle the `AccelerateWP` back.
+Both AccelerateWP and AccelerateWP Premium will be turned off.
+
+![](./images/awp/AWPDisable.png)
 
 This operation will:
 * disable AccelerateWP tab in users' cPanel interface
 * disable all AccelerateWP optimization suites
 * deactivate all optimization features for all users
 
+This operation will NOT:
+* cancel the subscription made by user in WHMCS or other billing
```

Comment:
Does it mean, that the hoster would still be billed for some users even when he stopped using AccelerateWP and has disallowed all users?
If not and this mean something like "subscription stays alive but deactivated for a period of time", may be we should formulate it a bit clearer?

Reply:
It means that "user's subscription stays alive and user will still be billed in billing system unless hoster manually terminates them"